cdap-app-fabric/src/main/java/org/apache/hadoop/yarn/util/ApplicationClassLoader.java

```diff
@@ -0,0 +1,168 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I am not sure if we need to keep this license header since we changed the class entirely. @henry-cask thoughts?